spec/support/idp_helpers.rb

```diff
@@ -40,6 +40,6 @@ def create_new_account
     code_words = acknowledge_personal_key
     expect(page).to have_content 'Welcome'
     puts "created account for #{email_address} with personal key: #{code_words.join('-')}"
-    email_address
+    [email_address, code_words.join('-')]
```

Comment:
Returning an array from this method feels a bit mysterious when in use. Perhaps a hash instead?